package-lock.json

```diff
@@ -347,12 +347,30 @@
       }
     },
     "node_modules/@bcrs-shared-components/certify": {
-      "version": "2.1.5",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/certify/-/certify-2.1.5.tgz",
-      "integrity": "sha512-+/6MRAHt1xRTcIauIg6lWDII9sLovl0rr2zAsV3+8zvm9+MVujNqC7mlG0CHI2Czl/tqT+CxpAhm/7m6pOk0+g==",
+      "version": "2.1.55",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/certify/-/certify-2.1.55.tgz",
```

Comment:
Requirement: Change ‘is’ to ‘are’
The Certify.vue in bcrs is already updated:
![image](https://github.com/user-attachments/assets/16f1147a-ad57-4f31-b9ca-f0e0bcfcabac)
However, the updated @bcrs-shared-components/certify version is 2.1.55, while edit UI is depending on 2.1.5
![image](https://github.com/user-attachments/assets/643afb18-dd77-40a6-bd24-3a8c196e1524)

Reply:
There have been a lot  of changes since this shared component was first imported! Please test it well in case there were some incompatible changes.

Reply:
Just tested it, looks good!